react/styles.css

```diff
@@ -1 +1,3 @@
-.container {}
+:global(.vtex__icon-spinner) {
+    position: aboslute;
```

Comment:
This can't be working 😆 
```suggestion
    position: absolute;
```

Reply:
I will improve the css. It's not last solution for this project.

Reply:
Where can I add a comment ?

Reply:
I wanna render my app only when the Stylesheets was created inside of card-form-ui. 
I'm currently using `<Spinner />` but I don't know if it's the best way to solve this UI problem. But maybe this change should not be part of this PR.

Reply:
You confused me with your comments lol. This is fine now that you fixed it 😆